ui/page/transaction/transactions_page.go

```diff
@@ -101,50 +115,42 @@ func (pg *TransactionsPage) OnNavigatedTo() {
 	go pg.scroll.FetchScrollData(false, pg.ParentWindow())
 }
 
-func (pg *TransactionsPage) sectionNavTab(gtx C) D {
-	var selectedTabDims D
-
-	return layout.Inset{
-		Top: values.MarginPadding20,
-	}.Layout(gtx, func(gtx C) D {
-		return pg.tabs.Layout(gtx, len(txTabs), func(gtx C, i int) D {
-			return layout.Stack{Alignment: layout.S}.Layout(gtx,
-				layout.Stacked(func(gtx C) D {
-					return layout.Inset{
-						Right:  values.MarginPadding24,
-						Bottom: values.MarginPadding14,
-					}.Layout(gtx, func(gtx C) D {
-						return layout.Center.Layout(gtx, func(gtx C) D {
-							lbl := pg.Theme.Label(values.TextSize16, txTabs[i])
-							lbl.Color = pg.Theme.Color.GrayText1
-							if i == pg.selectedTabIndex {
-								lbl.Color = pg.Theme.Color.Primary
-								selectedTabDims = lbl.Layout(gtx)
-							}
+// initWalletSelector is used by the tx history page, for wallet selection.
+func (pg *TransactionsPage) initWalletDropdown() {
+	pg.showLoader = true
+	pg.assetWallets = pg.WL.AllSortedWalletList()
+	if pg.tab.SelectedSegment() != values.String(values.StrTxOverview) {
+		pg.assetWallets = pg.WL.SortedWalletList(utils.DCRWalletAsset)
+	}
 
-							return lbl.Layout(gtx)
-						})
-					})
-				}),
-				layout.Stacked(func(gtx C) D {
-					if i != pg.selectedTabIndex {
-						return D{}
-					}
+	items := []cryptomaterial.DropDownItem{}
+	for _, wal := range pg.assetWallets {
+		item := cryptomaterial.DropDownItem{}
+		switch wal.GetAssetType() {
+		case utils.DCRWalletAsset:
+			item.Icon = pg.Theme.Icons.DCR
+		case utils.LTCWalletAsset:
+			item.Icon = pg.Theme.Icons.LTC
+		case utils.BTCWalletAsset:
+			item.Icon = pg.Theme.Icons.BTC
+		}
+		item.Text = wal.GetWalletName()
+		items = append(items, item)
+	}
 
-					tabHeight := gtx.Dp(values.MarginPadding2)
-					tabRect := image.Rect(0, 0, selectedTabDims.Size.X, tabHeight)
+	pg.walletDropDown = pg.Theme.DropDown(items, values.WalletsDropdownGroup, 0)
+	pg.selectedWallet = pg.assetWallets[pg.walletDropDown.SelectedIndex()]
+	pg.showLoader = false
```

Comment:
What is achieved by setting this to true at the beginning of the method and then changing it to false at the end? It appears this pg.showLoader property is meant to influence the UI layout but UI layout doesn't happen at any point between the start and the end of this method. Changing the pg.showLoader value at the start and end of this method seems moot to me.